core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationLoader.java

```diff
@@ -107,20 +107,20 @@ static Configuration load(Path path) {
     }
 
     private static Path getConfigurationFilePath(File[] files) {
-        if (files.length == 1) {
-            final File configFile = files[0];
-            logger.info("Using configuration from " + configFile.getName());
-            return configFile.toPath();
+        if (files.length == 2) {
+            logger.warn(
+                "Found multiple config files with supported names: " + SMART_TESTING_YAML + ", " + SMART_TESTING_YML);
```

Comment:
Should we warn or fail here? Or maybe `warn`, but decide which one we use? Wondering if `findAny` always returns the same result.